docs/content/latest/architecture/docdb-sharding/tablet-splitting.md

```diff
@@ -12,67 +12,74 @@ isTocNested: true
 showAsideToc: true
 ---
 
-Tablet splitting enables changing the number of tablets at runtime. This enables *re-shrading* the existing data in the cluster.
-
 ## Overview
 
-There are a number of scenarios where this is useful.
+Tablet splitting enables *resharding* existing data in the cluster by changing the number of tablets at runtime. Here are some of the scenarios where this is useful.
```

Comment:
technically, presplitting doesn't fit in the category of changing things at runtime, as it's done in advance

Reply:
right, that's what i meant when i said in slack that this page started off intending to discuss only runtime changes, but then added the presplitting and manual mechanisms. I'll spend a couple of minutes and make this clearer, but thought i'd get stuck here too long.